apps/cbportal/base/CBGrid/.macros/page/0accessnode/3_0accessnode.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import json
+import datetime
+from jose import jwt as jose_jwt
+import ast
+
+def main(j, args, params, tags, tasklet):
+    page = args.page
+    counter = """
+<script>
+window.onload=function(){
+var counter = %s;
+var interval = setInterval(function() {
+    counter--;
+    document.getElementById('counter').innerText = "The following data will be expired in " + counter + " seconds if you didn't use it"
+    if (counter <= 0) {
+        document.getElementById('counter').innerText = "this data is no longer valid"
+    }
+}, 1000);
+}
+</script>
+    """
+    remote = args.requestContext.params.get('ip')
+    jwt = args.requestContext.env['beaker.session'].get('jwt')
+    if jwt:
```

Comment:
i suggest to make a 0-access client which we use in all macros, instead of repeating the logic for this stuff 
and then in macro we do some thing like this 
```
access_client.list_sessions()
```